config/prod/prow/pj-on-kind.sh

```diff
@@ -27,7 +27,13 @@ set -o pipefail
 PROW_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 
 # Download prow core config from prow
-CONFIG_YAML="$(mktemp)"
+if [[ $IS_OSX ]]; then
+  # On OS X, the file has to be under /private other it cannot be mounted by the container.
+  # See https://stackoverflow.com/questions/45122459/docker-mounts-denied-the-paths-are-not-shared-from-os-x-and-are-not-known/45123074
+  CONFIG_YAML="/private"$(mktemp)
```

Comment:
nit, we don't need this condition checking

Reply:
On my linux machine I don't have the `/private` folder, and I got an error if the temp file name has the `/private` prefix, so this check is needed AFAICT.

Reply:
Can you `mkdir -p /private` in this case?

Reply:
nvm, permission denied